pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/routing/Routing.kt

```diff
@@ -28,7 +28,10 @@ fun Application.brevbakerRouting(authenticationNames: Array<String>, latexCompil
             }
 
             route("etterlatte") {
-                etterlatteRouting(latexCompilerService)
+                letterRoutes(
```

Comment:
Har vi integrasjonstester for etterlatte?

Reply:
Bør koordinere litt med etterlatte, og poste PR til de før du merger til main.

Reply:
Vi har testar på sjølve malane deira, men ikkje på infra-koden rundt. Heldigvis ganske enkelt å teste manuelt frå Gjenny, men det er jo også ein fordel med denne endringa.

Reply:
Får kjøre det ut i q2 for en smoke-test iallefall :)

Reply:
Testa ei stund no, og testa litt frå Gjenny sjølv, alt fint i både loggar og gui og alt.